thirdparty/build-thirdparty.sh

```diff
@@ -514,14 +514,29 @@ build_rocksdb() {
     export CFLAGS=$OLD_FLAGS
 }
 
+# sasl
+build_sasl() {
+    check_if_source_exist $SASL_SOURCE
+
+    cd $TP_SOURCE_DIR/$SASL_SOURCE
+    sh ./autogen.sh
+    ./configure --prefix=$TP_INSTALL_DIR  --enable-static=yes
+    make -j$PARALLEL
+    make install
+    cd $TP_INSTALL_DIR/lib
+    ls | grep sasl | grep so | xargs rm -rf
+    cd $TP_INSTALL_DIR/lib/sasl2
+    find . | grep so | xargs rm -rf
```

Comment:
```suggestion
    ./autogen.sh --prefix=$TP_INSTALL_DIR  --enable-static=yes --enable-shared=no
    make -j$PARALLEL
    make install
```

Reply:
also need to point the openssl install location and gssapi location, or disable it explicitly.